connector/scrapers/kutis_scraper.py

```
@@ -0,0 +1,130 @@
+from bs4 import BeautifulSoup
+
+from connector.clients import KutisClient
+from connector.common import HTTPMethod
+from connector.constants import *
```

Ruff (0.11.9)

`from connector.constants import *` used; unable to detect undefined names

(F403)